cmd/terramate/cli/cloud.go

```diff
@@ -392,6 +326,93 @@ func (c *cli) cloudInfo() {
 	c.cloud.output.MsgStdOutV("next token refresh in: %s", time.Until(c.cred().ExpireAt()))
 }
 
+func (c *cli) reviewRequest(normalizedRepo string) (*cloud.DeploymentReviewRequest, string) {
+	logger := log.With().
+		Str("normalized-repo", normalizedRepo).
+		Str("head-commit", c.prj.headCommit()).
```

Comment:
afaics this introduced the third way to name fields as we use camelCase and underscores already?

```suggestion
		Str("normalized_repository", normalizedRepo).
		Str("head_commit", c.prj.headCommit()).
```